packages/twitch/src/API/Helix/HypeTrain/HypeTrainEvent.ts

```diff
@@ -24,6 +24,11 @@ export class HypeTrainEvent {
 	/** @private */
 	constructor(/** @private */ protected _data: HypeTrainEventData, client: ApiClient) {
 		this._client = client;
+		this._data.lastContribution = new HypeTrainContributor(_data.lastContribution);
```

Comment:
Please don't overwrite raw data with class instances. Instead, convert in the getters, as I said.